scripts/pin-to-cluster.sh

```diff
@@ -41,6 +41,7 @@ root_cid=$(ipfs-cluster-ctl \
     --host $HOST \
     --basic-auth $CLUSTER_USER:$CLUSTER_PASSWORD \
     add --quieter \
+    --local \
```

Comment:
Does `--local` require there to be a locally running ipfs daemon? We don't have one of those in this container.

Reply:
No, just means that the cluster peer receving the files does should add them locally itself, rather than broadcasting the blocks to all destinations.

Reply:
Once it's done, it will pin and bitswap will copy the things to the right places (possibly faster)

Reply:
just out of curiosity, why not make that behaviour the default?